content/spin/contributing-docs.md

```diff
@@ -405,6 +405,21 @@ $ bart check content/about.md
 ✅ content/about.md
 ```
 
+If you are [using shortcodes](/bartholomew/shortcodes) and need to tell the `bart check` command where to find them (i.e. if you have your reusable snippets, written in `.rhai` files, tucked away in a relative directory called `my-shortcodes`), you can use the `--shortcodes` option. For example:
```

Comment:
This can be simplified for this repo by saying "always pass `--shortcodes ./shortcodes`".  People contributing to the Fermyon docs aren't going to have a different shortcodes directory, and there's no "if" about whether the site uses shortcodes.

This _would_ be useful to add to the Bartholomew docs themselves though!